azure/azure-pr-teardown-pipeline.yml

```diff
@@ -50,10 +50,10 @@ jobs:
           make init
           # tf_vars has the client id, secret, and keycloak env required by default so we pass null values here to
           # prevent terraform from hanging waiting for an input for these values
-          terraform workspace select $(WORKSPACE) && make init && terraform destroy -auto-approve -force /
-          -var="client_id=$(client_id)" -var="client_secret=$(client_secret)" /
-          -var="keycloak_environment=$(keycloak_environment)" && /
-          terraform workspace select default && terraform workspace delete $(WORKSPACE)
+          terraform workspace select $(WORKSPACE)
```

Comment:
How is this working Almas? You can't be in the `WORKSPACE` while destroying it. Should it be `default`? 
Also the comment above, I know you didn't wrote it but do you know what that means? Terraform has the `-input=false` option.

Reply:
We first selecting that workspace and doing
 make init and make destroy
 In make destroy, we are selecting default workspace